### DISCUSSION TITLE
Is it possible to use SQS with a custom API endpoint.

### DISCUSSION THREAD

**Discussion by maeln** [celery/celery#8735] [Q&A]
Hi,

I read the page about SQS and Celery ( https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/sqs.html ) and I notice that it wasn't mentioned if it is possible to use a different endpoint URL for the SQS API.
This is required if you want to use another cloud provider that provide a SQS compatible API, and also to test locally using a queue that expose a SQS compatible API like ElasticMQ.
For exemple, in `boto3` you can use the property `endpoint_url` to set it up: 
```python
sqs_session = boto3.Session()
sqs_client = sqs_session.client(
    service_name="sqs",
    region_name="fr-par",
    aws_access_key_id=AWS_ACCESS_KEY,
    aws_secret_access_key=AWS_SECRET_KEY,
    endpoint_url="https://sqs.mnq.fr-par.scaleway.com",
)
```

Is it possible by adding a specific key in `broker_transport_options` or by setting up an environment variable like `AWS_ENDPOINT_URL_SQS` ?

**Comment by avinashgbhosale** [comment]
Hi team, does anyone have solution for this. I am using local stack for providing sqs. Please help me for configuring local endpoint url
